### DISCUSSION TITLE
MKL usage

### DISCUSSION THREAD

**Discussion by YahyaFi** [osqp/osqp#642] [General]
i am trying to leverage the mkl library but i am not familiar with cmake enough and getting error.
i have already the mkl libs and everything related to mkl but unfortunately i cannot link is there a simple procedure to make it work please.
Thanks

**Comment by imciner2** [comment]
Which version of OSQP are you trying to use, and what is the error that is being reported?

**Comment by YahyaFi** [comment]
Thank you for your response. I tried to make it manually using visual studio and it is working is there an automatic way to do it.
I m using the head of the master branch.

I wanna ask as well if it is meant to enable the flag  OSQP_ENABLE_DERIVATIVES when using mkl mode I might think it is not supported.
Thank you.
